components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -918,7 +918,7 @@
 <micro_nccons         microphys="mg2">  100.D6      </micro_nccons>
 <micro_nicons         microphys="mg2">  0.1D6       </micro_nicons>
 <micro_mincdnc        microphys="mg2">  -999.       </micro_mincdnc>
-
+<p3_mincdnc           microphys="p3">  -999.        </p3_mincdnc> 
```

Comment:
why is this part of this PR? maybe, add these changes to the description?

Reply:
I don't recall changing this file...

Reply:
The log shows for the first two commits

```
Yunpeng Shan authored and [bartgol](https://github.com/E3SM-Project/E3SM/commits?author=bartgol) committed yesterday
```
I don't know how this ended in the PR. I'll remove those commit and force push a clean version.